### PR TITLE
auth NOTIFY: log names of NSes we could not resolve

### PR DESCRIPTION
**Pull request by Habbie**
### Checklist

I have:
- [x] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [x] compiled this code
- [x] tested this code
- [ ] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added or modified regression test(s)
- [ ] added or modified unit test(s)
- [ ]  checked that this code was merged to master

### CODE REVIEW

File: pdns/mastercommunicator.cc [hunk 1]
```diff
@@ -58,15 +58,16 @@ void CommunicatorClass::queueNotifyDomain(const DomainInfo& di, UeberBackend* B)
...
       if(nsips.empty())
-        g_log<<Logger::Warning<<"Unable to queue notification of domain '"<<di.zone<<"': nameservers do not resolve!"<<endl;
+        g_log<<Logger::Warning<<"Unable to queue notification of domain '"<<di.zone<<"' to nameserver '"<<dnsj<<"': nameserver does not resolve!"<<endl;
       else
         for(const auto & nsip : nsips) {
           const ComboAddress caIp(nsip, 53);
           if(!d_preventSelfNotification || !AddressIsUs(caIp)) {
             if(!d_onlyNotify.match(&caIp))
-              g_log<<Logger::Notice<<"Skipped notification of domain '"<<di.zone<<"' to "<<j<<" because it does not match only-notify."<<endl;
+              g_log<<Logger::Notice<<"Skipped notification of domain '"<<di.zone<<"' to "<<dnsj<<" because it does not match only-notify."<<endl;
```
Comment (mind04) [inline pdns/mastercommunicator.cc:70]: Please, mention the mismatched ip in this log message.

Comment (Habbie) [reply pdns/mastercommunicator.cc]: makes sense, done

**Review by mind04 — APPROVED** [review]

File: pdns/mastercommunicator.cc [hunk 2]
```diff
@@ -58,15 +58,16 @@ void CommunicatorClass::queueNotifyDomain(const DomainInfo& di, UeberBackend* B)
       nsset.insert(getRR<NSRecordContent>(rr.dr)->getNS().toString());
 
     for(const auto & j : nsset) {
-      vector<string> nsips=fns.lookup(DNSName(j), B);
+      auto dnsj = DNSName(j);
```
Comment (pieterlexis) [inline pdns/mastercommunicator.cc:61]: when you're here anyway. Maybe rename `j` to something better?

Comment (Habbie) [reply pdns/mastercommunicator.cc]: I did something even better :)

**Review by pieterlexis — APPROVED** [review]

**Pull request merged**